src/expr.h

```diff
@@ -149,7 +149,7 @@ class Expr
   inline void dec(bool dec_kids = true)
   {
     int ref = getrefcnt();
-    ref = ref - 1;
+    ref = ref < 4194303 ? ref - 1 : ref;
```

Comment:
Can we define this constant somewhere? It seems error prone to repeat it everywhere.